astropy/tests/runner.py

```diff
@@ -463,8 +463,9 @@ def remote_data(self, remote_data, kwargs):
         remote_data : {'none', 'astropy', 'any'}, optional
             Controls whether to run tests marked with @pytest.mark.remote_data. This can be
             set to run no tests with remote data (``none``), only ones that use
-            data from http://data.astropy.org (``astropy``), or all tests that
```

Comment:
Wait a minute... http://data.astropy.org still works.

Reply:
Yes, but it redirects to https://www.astropy.org/astropy-data/

Reply:
That might change in the future... 👀

Reply:
OK, should I close this then?

Reply:
Yeah... Sorry for the inconvenience!